src/Microsoft.Health.Dicom.Core/Features/CustomTag/GetCustomTagsService.cs

```diff
@@ -0,0 +1,46 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Health.Dicom.Core.Messages.CustomTag;
+
+namespace Microsoft.Health.Dicom.Core.Features.CustomTag
+{
+    public class GetCustomTagsService : IGetCustomTagsService
+    {
+        private readonly ICustomTagStore _customTagStore;
+
+        public GetCustomTagsService(ICustomTagStore customTagStore)
+        {
+            EnsureArg.IsNotNull(customTagStore, nameof(customTagStore));
+
+            _customTagStore = customTagStore;
+        }
+
+        public async Task<GetCustomTagResponse> GetCustomTagAsync(string tagPath, CancellationToken cancellationToken = default)
```

Comment:
"CancellationToken cancellationToken = default"
Default value could be specified on interface only